routes/article.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express')
+const router = express.Router()
+
+router.get('articles/article/:id', (req, res) => {
```

Comment:
You need a `/` at the start of the path for it to work.
```suggestion
router.get('/articles/article/:id', (req, res) => {
```